test/jdk/javax/swing/SwingWorker/TestDoneBeforeDoInBackground.java

```diff
@@ -64,14 +68,46 @@ protected String doInBackground() throws Exception {
 
             @Override
             protected void done() {
-                if (!doInBackground) {
-                    throw new RuntimeException("done called before doInBackground");
+                if (!doInBackgroundFinished.get()) {
+                    throw new RuntimeException("done called before " +
+                                             " doInBackground is finished");
```

Comment:
```suggestion
                    throw new RuntimeException("done called before " +
                                               "doInBackground is finished");
```

Add two spaces before the quote and to align it to the one on the line above; remove one space after the quote so that the error message doesn't have two consecutive spaces.